database/postgres/subscription.go

```diff
@@ -121,6 +122,40 @@ const (
     AND id > $1
     AND project_id = $2
     AND deleted_at is null
+    ORDER BY id LIMIT $3`
+
+	loadAllSubscriptionsConfiguration = `
+    select name, id, type, project_id, endpoint_id, function, updated_at,
+    filter_config_event_types AS "filter_config.event_types",
+    filter_config_filter_headers AS "filter_config.filter.headers",
+	filter_config_filter_body AS "filter_config.filter.body"
+    from convoy.subscriptions
+    where id > $1
+    AND project_id = $2
+    AND deleted_at is null
+    ORDER BY id LIMIT $3`
+
+	fetchUpdatedSubscriptions = `
+    select name, id, type, project_id, endpoint_id, function, updated_at,
+    filter_config_event_types AS "filter_config.event_types",
+    filter_config_filter_headers AS "filter_config.filter.headers",
+	filter_config_filter_body AS "filter_config.filter.body"
+    from convoy.subscriptions
+    where updated_at > $4
+    AND id > $1
+    AND project_id = $2
+    AND deleted_at is null
+    ORDER BY id LIMIT $3`
+
+	fetchDeletedSubscriptions = `
+    select name, id, type, project_id, endpoint_id, function, updated_at,
+    filter_config_event_types AS "filter_config.event_types",
+    filter_config_filter_headers AS "filter_config.filter.headers",
+	filter_config_filter_body AS "filter_config.filter.body"
+    from convoy.subscriptions
+    where (deleted_at > $4 AND deleted_at is not null)
+    AND id > $1
+    AND project_id = $2
```

Comment:
@subomi please make a base fetch constant query, that way we can avoid any one of them just randomly changing.

Reply:
@danvixent I'll defer this for a later refactor.